packages/forms/signals/test/node/api/validators/util.spec.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {ensureCustomValidationResult} from '../../../../src/api/validators/util';
+import {
+  customError,
+  minError,
+  ValidationError,
+  ValidationErrorWithField,
+} from '../../../../src/api/validation_errors';
+import {FieldTree} from '../../../../src/api/types';
+import {addDefaultField} from '../../../../src/field/validation';
+
+describe('validators utils', () => {
+  describe('makeValidationResultCustomIfNeeded', () => {
```

Comment:
nit: is this meant to be
```suggestion
  describe('ensureCustomValidationResult', () => {
```